README.md

```diff
@@ -177,6 +177,10 @@ The will automatically create and run the following containers:
 * **cachito-worker** - the Cachito Celery worker. This container is also responsible for configuring
   Nexus at startup.
 * **db** - the Postgresql database used by the Cachito REST API.
+* **flower** - the [Flower](https://flower.readthedocs.io/en/latest/) instance for monitoring Celery 
```

Comment:
Could we say `monitoring and managing Celery` instead of `monitoring Celery tasks and administrating the Celery container (cachito-worker)`?